ts/util/releaseFeature.ts

```diff
@@ -118,12 +119,11 @@ export const ReleasedFeatures = {
   isDisappearMessageV2FeatureReleasedCached,
 };
 
-// DO NOT MERGE
-// TODO Remove before PR
+// TODO DO NOT MERGE Remove after QA
```

Comment:
we need to remember this

Reply:
I've made a note in my journal for this.